src/aat/resources/features/F-105/F-105.feature

```diff
@@ -0,0 +1,201 @@
+    And a user [Olawale - with an active solicitor profile and valid User ID],
+    And a successful call [by Richard to create a case - C1] as in [F-105_Prerequisite_Case_Creation_Call_for_Case_Assignment],
+    When a request is prepared with appropriate values,
+    And the request [is made by Dil for 2 assignments each containing the Case ID of C1 and a proper Case Role CR-1]
+    And the request [contains the User ID of Olawale in one entry but no User ID in the other]
+    And it is submitted to call the [Add Case-Assigned Users and Roles] operation of [CCD Data Store api],
+    Then a negative response is received,
+    And the response has all the details as expected.
+    And a call [to verify Olawale's reception of the role CR-1 over the case C1] will get the expected response as in [S-105.4_Get_Case_Roles_for_Case_C1].
+
+  # RDM-8606/8806 AC-5
+  @S-105.5
+  Scenario: Must return an error response for a malformed User ID Provided
+    Given an appropriate test context as detailed in the test data source,
+    And a user [Richard - who can create a case],
+    And a user [Dil - who is to add some case role assignment for a case],
+    And a user [Olawale - with an active solicitor profile and valid User ID],
+    And a successful call [by Richard to create a case - C1] as in [F-105_Prerequisite_Case_Creation_Call_for_Case_Assignment],
+    When a request is prepared with appropriate values,
+    And the request [is made by Dil for 2 assignments each containing the Case ID of C1 and a proper Case Role CR-1]
+    And the request [contains the User ID of Olawale in one entry and a malformed User ID in the other]
+    And it is submitted to call the [Add Case-Assigned Users and Roles] operation of [CCD Data Store api],
+    Then a negative response is received,
+    And the response has all the details as expected.
+    And a call [to verify Olawale's reception of the role CR-1 over the case C1] will get the expected response as in [S-105.5_Get_Case_Roles_for_Case_C1].
+
+  # RDM-8606/8806 AC-6
+  @S-105.6
+  Scenario: Must return an error response when the request is made from an un-authorised application
+    Given an appropriate test context as detailed in the test data source,
+    And a user [Richard - who can create a case],
+    And a successful call [by Richard to create a case - C1] as in [F-105_Prerequisite_Case_Creation_Call_for_Case_Assignment],
+    And a user [Dil - with an active profile],
+    And a user [Olawale - who is not a privileged user and is calling from an un-authorised application],
+    When a request is prepared with appropriate values,
+    And the request [is made by Olawale with the Case ID of C1 & Dil's User ID and a proper Case Role CR-1],
+    And it is submitted to call the [Add Case-Assigned Users and Roles] operation of [CCD Data Store api],
+    Then a negative response is received,
+    And the response has all the details as expected.
+
```

Comment:
a call [to verify that Dil hasn't received the role CR-1 over the case C1] ...

Reply:
I have added it along with corresponding verify JSON.